docs/operator-bundle/testing-and-releasing.md

```diff
@@ -0,0 +1,181 @@
+# Testing and releasing the operatorhub bundle for ASO
+
+This is a guide to producing and testing the operator bundle after an ASO release.
+
+## Test upgrading from the old bundle to the new one
+
+### Before you begin
+
+You'll need:
+
+* A previous operator hub bundle (for me this is `quay.io/operatorhubio/azure-service-operator:v0.37.0`)
+* A newly published operator image that needs wrapping in an operator bundle (eg. `mcr.microsoft.com/k8s/azureserviceoperator:1.0.22275`)
+* A cluster with OLM installed (but without ASO installed). Installing OLM:
```

Comment:
Does this belong on the end of something in the prerequisites list?

Reply:
Oh, I see, there are instructions interspersed in the bullet list. 🤔

Reply:
The rationale for having these setup instructions in the "before you begin" section is that you don't need to do them every time.

Reply:
I've done that.